dockers/docker-fpm/Dockerfile.j2

```diff
@@ -2,7 +2,7 @@ FROM docker-base
 
 RUN apt-get update
 
-RUN apt-get install -y libdbus-1-3 libdaemon0 libjansson4
+RUN apt-get install -y libdbus-1-3 libdaemon0 libjansson4 python-lxml python-jinja2 python-netaddr python-ipaddr python-yaml
```

Comment:
I vote to keep dev packages out of base image. In concept, they are part of our deliverable and expected updating/adding/remove frequently then a base image.

---
In reply to: [96108784](https://github.com/Azure/sonic-buildimage/pull/190#discussion_r96108784) [](ancestors = 96108784)